content/python/encyclopedia/classes-in-python.md

```diff
@@ -0,0 +1,47 @@
+<!--
+---
+Subjects:
+  - "What are Classes in Python?"
```

Comment:
The "Subjects" field is being used as an additional tagging system. Please pick tags from Subjects that are already listed on Author here: https://author.codecademy.com/browse/categories.